### ISSUE TITLE
Assets and port 8080

### ISSUE THREAD

**Issue by tomazbracic** [kittoframework/kitto#13]
Hi, 

I tried and would still like to switch port 8080 which is used to serve JS, assets and more. 
Who is serving this? Nodejs? How can I change it? It is in conflict with my other non-kittio "stuff" for which I use port 8080.

Best,
Tomaz

**Comment by davejlong** [comment]
This actually seems to not be changeable at this time. The asset dev server, run by Node.js, uses an environment variable (`PORT`) as you can see from [webpack.config.js#L81](https://github.com/kittoframework/kitto/blob/master/installer/templates/new/webpack.config.js#L81), but the router is statically set to 8080 [router.ex#L4](https://github.com/kittoframework/kitto/blob/master/lib/kitto/router.ex#L4).

**Comment by zorbash** [comment]
We should allow the user to change the [asset dev server](https://webpack.github.io/docs/webpack-dev-server.html) port. The problem in our case as @davejlong noted, is the hardcoded url in the router, which should take the port env variable into consideration.

We should also change the webpack.config not to use `PORT`, which is assigned by heroku (see: #5) for the kitto server (see also https://devcenter.heroku.com/articles/runtime-principles#web-servers).

**Comment by zorbash** [comment]
I've started some work for this on https://github.com/kittoframework/kitto/tree/configurable-asset-server-address

**Comment by zorbash** [comment]
70656bb608 allows changing webpack-dev-server port using

``` elixir
# File: config/config.exs 
config :kitto, assets_port: 1337
```

**Comment by tomazbracic** [comment]
This will be in next release? Or?....

**Comment by zorbash** [comment]
@tomazbracic it's now released in v0.2.3.
